src/components/ProductDetail/ProductDetail.scss

```diff
@@ -0,0 +1,95 @@
+.lds-spinner {
+    color: official;
+    display: flex;
+    justify-content: center;
+    align-items: center;   
+    position: relative;
+    max-width: 845px;
+    height: 180px;
+   
+    div {
+       transform-origin: 40px 40px;
+       animation: lds-spinner 1.2s linear infinite;
+    
+       &:after {
+          content: " ";
+          display: block;
+          position: absolute;
+          top: 3px;
+          left: 37px;
+          width: 6px;
+          height: 18px;
+          border-radius: 20%;
+          background: black;
+       }
+ 
+       &:nth-child(1) {
+          transform: rotate(0deg);
+          animation-delay: -1.1s;
+       }
+ 
+       &:nth-child(2) {
+          transform: rotate(30deg);
+          animation-delay: -1s;
+       }
+       
+       &:nth-child(3) {
+          transform: rotate(60deg);
```

Comment:
https://sass-lang.com/documentation/at-rules/control/for tutaj tego można użyć

Daj mi znać jak będziesz chciała podpowiedź jak to ma wyglądać to napiszę funkcję :)